test/TestLibrary/acutest.h

```diff
@@ -1150,7 +1150,8 @@ acutest_run_(const struct acutest_test_* test, int index, int master_index)
                     case SIGSEGV: signame = "SIGSEGV"; break;
                     case SIGILL:  signame = "SIGILL"; break;
                     case SIGTERM: signame = "SIGTERM"; break;
-                    default:      sprintf(tmp, "signal %d", WTERMSIG(exit_code)); signame = tmp; break;
+                    //default:      sprintf(tmp, "signal %d", WTERMSIG(exit_code)); signame = tmp; break;
+                    default:      snprintf(tmp, 32, "signal %d", WTERMSIG(exit_code)); signame = tmp; break;
```

Comment:
pewny jesteś że tu powinno być to snprintf ? Bo tam niżej kilkanaście lini ktoś skorzystał z `_snprintf`